multi-node/generic/controller-install.sh

```diff
@@ -27,13 +30,16 @@ export K8S_SERVICE_IP=10.3.0.1
 # This same IP must be configured on all worker nodes to enable DNS service discovery.
 export DNS_SERVICE_IP=10.3.0.10
 
+# Whether to use Calico for Kubernetes network policy.
+export USE_CALICO=true
```

Comment:
In the generic scripts we should default to `USE_CALICO=false` for now, as people use these for deployments to cloud-providers not otherwise supported by this tool. I'm fine with the vagrant single/multinode installs flipping this to true, but this file should default to off so behavior doesn't change for existing users.